command/v6/service_access_command_test.go

```diff
@@ -173,30 +264,52 @@ var _ = Describe("service-access Command", func() {
 				})
 			})
 
-			Describe("table", func() {
-				var brokers []v2action.ServiceBrokerSummary
+			Describe("tabular output", func() {
+				When("there are multiple brokers", func() {
+					var brokers []v2action.ServiceBrokerSummary
 
-				BeforeEach(func() {
-					brokers = generateBrokers(2)
-					fakeActor.GetServiceBrokerSummariesReturns(generateBrokers(2), v2action.Warnings{"warning"}, nil)
+					BeforeEach(func() {
+						brokers = generateBrokers(2)
```

Comment:
Why is generateBrokers called on this line and then again on the line below? Isn't the output going to be identical?